### DISCUSSION TITLE
Qual Static Site Generator vocês utilizam?

### DISCUSSION THREAD

**Discussion by dionarodrigues** [frontendbr/forum#905] [Perguntas e Respostas]
Fala galera! 

Pensando em criar um novo site para mim, comecei a pesquisar qual seria a melhor alternativa de se fazer isso sem utilizar um CMS, como o WordPress por exemplo. E encontrei algumas alternativas através dos Static Site Generators: [Docpad](https://docpad.org/), [Harp ](http://harpjs.com/)e [Jekyll](https://jekyllrb.com/), for example.

Com essa descoberta, são dois pontos que estou tentando entender:

**Qual a diferença (vantagens/desvantagens) de não criar um site estático utilizando apenas Template Engines (como o Pug, for example)?**  

Vejam abaixo o motivo dessa minha primeira dúvida: 

Criei o [Iceberg Boilerplate](https://github.com/diogorodrigues/iceberg-boilerplate) e sempre o utilizo em meus projetos pessoais. Esse boilerplate me atende bem, porém, começa a ficar lento conforme a quantidade de arquivos Pug vai aumentando. Mas creio que não deve ser essa a principal diferença entre um Template Engine e um Static Site Generator, certo?

Minha segunda dúvida é **dentre os Static Site Generators quais vocês costumam usar e por quê?**

Vlw!!

**Comment by felipefialho** [comment]
O que mais curti até hoje foi o [harp](http://harpjs.com/).

É bastante flexivel, simples de entender e integra com praticamente qualquer _template enginer_ e _css pre-processor_, além do suporte a Markdown.

**Comment by mahenrique94** [comment]
Utilizo o Jekyll e Hugo, ambos são ótimos, sempre me atenderam bem.

**Comment by felipemfp** [comment]
Cara, dá uma olhada no [Gatsby](https://github.com/gatsbyjs/gatsby). É sensacional! E vem crescendo muito.

**Comment by expalmer** [comment]
Eu uso [Metalsmith](http://www.metalsmith.io/) no [meu blog](http://expalmer.github.io/)
